samples/deployment/aws/deploy_and_destroy.yaml

```diff
@@ -0,0 +1,130 @@
+- name: "My AWS infrastructure"
+  hosts: localhost
+  gather_facts: no
+  vars:
+    ftd_build_id: "6.4.0-1304"
+    ec2_instance_type: "c4.xlarge"
+    ansible_ssh_private_key_name: "ngfwv-prod-temp-kp.pem"
```

Comment:
I think it should be `ansible_ssh_keypair_name`. `ec2_key` creates a key pair that contains a private key, so it's incorrect to name it `private_key_name` and add `.pem` extension.

Reply:
Right, currently it creates a key pair with ".pem" in its name which is incorrect.

Reply:
fixed